v22.2/show-backup.md

```diff
@@ -190,6 +190,33 @@ movr          | public             | vehicles                   | table       |
 . . .
 ~~~
 
+### Show a locality-aware backup
+
+To view a [locality-aware backup](take-and-restore-locality-aware-backups.html), pass locality-aware backup URIs to `SHOW BACKUP`:
+
+{{site.data.alerts.callout_info}}
+`SHOW BACKUP` can display backups taken with the [`incremental_location`](show-backup.html#show-a-backup-taken-with-the-incremental-location-option) option **or** for locality-aware backups, but not for locality-aware backups taken with the `incremental_location` option.
+{{site.data.alerts.end}}
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> SHOW BACKUP FROM LATEST in ('s3://{bucket name}/locality?AWS_ACCESS_KEY_ID={placeholder}&AWS_SECRET_ACCESS_KEY={placeholder}&COCKROACH_LOCALITY=default', 's3://{bucket name}/locality?AWS_ACCESS_KEY_ID={placeholder}&AWS_SECRET_ACCESS_KEY={placeholder}&COCKROACH_LOCALITY=region%3Dus-west');
+~~~
+
+~~~
```

Comment:
given that we don't display any locality specific data in the show backup results, is there a reason to display the results from this query? Or rather, should we explicitly say that no locality aware data appears in the results?

Reply:
Even though there is no locality specific data being shown in the results, we will display the query results on the page to keep the same structure as the other examples

Reply:
Sounds good!